core/src/http_utils.rs

```diff
@@ -0,0 +1,40 @@
+use axum::Json;
```

Comment:
we can probably have that in `src/util.rs` ?

Reply:
Fair 👍